packages/fe/store/dataset.js

```diff
@@ -107,8 +39,8 @@ const actions = {
         }
       })
       const dataset = response.data.payload
-      dispatch('getCidList', { datasetId: dataset._id })
       commit('SET_DATASET', { dataset })
+      await dispatch('getCidList', metadata)
```

Comment:
Let's make this call (getCidList) strictly client-side (in the mounted hook on the singular page) so that it doesn't hold up loading of all the other dataset info.